src/formatDate.js

```diff
@@ -7,8 +7,36 @@
  *
  * @returns {string}
  */
-function formatDate(date, fromFormat, toFormat) {
-  // write code here
+function convertToYYYY(year) {
+  return parseInt(year) < 30 ? '20' + year : '19' + year;
+}
+
+function convertToYY(year) {
+  return year.slice(-2);
 }
```

Comment:
The `convertToYY` function does not handle the case when the year is less than 2000. In this case, the result will be incorrect. You should check if the year is less than 2000 and return the last two digits of the year if it is.